path_config.go

```diff
@@ -37,13 +38,45 @@ clears the current setting. Accepts a Go duration format string.`,
 			logical.UpdateOperation: &framework.PathOperation{
 				Callback: b.upgradeCheck(b.pathConfigWrite()),
 				Summary:  "Configure backend level settings that are applied to every key in the key-value store.",
+				Responses: map[int][]framework.Response{
+					http.StatusNoContent: {{
+						Description: http.StatusText(http.StatusNoContent),
+					}},
+				},
 			},
 			logical.CreateOperation: &framework.PathOperation{
 				Callback: b.upgradeCheck(b.pathConfigWrite()),
+				Responses: map[int][]framework.Response{
+					http.StatusNoContent: {{
+						Description: http.StatusText(http.StatusNoContent),
+					}},
+				},
 			},
 			logical.ReadOperation: &framework.PathOperation{
 				Callback: b.upgradeCheck(b.pathConfigRead()),
 				Summary:  "Read the backend level settings.",
+				Responses: map[int][]framework.Response{
+					http.StatusOK: {{
+						Description: http.StatusText(http.StatusOK),
+						Fields: map[string]*framework.FieldSchema{
+							"max_versions": {
+								Type:        framework.TypeInt,
+								Description: "The number of versions to keep for each key. Defaults to 10",
```

Comment:
should this be
```suggestion
								Description: "The number of versions to keep for each key. Defaults to 10",
								Default: 10,
```
?

Reply:
I don't think `Default` makes too much sense in the context of a response. For a request, if a value is not set it will default to the default value. For response I don't know how I would interpret it. How about this instead:

```suggestion
								Description: "The number of versions to keep for each key.",
```